test/api_integration/config.js

```diff
@@ -43,7 +43,6 @@ export default async function ({ readConfigFile }) {
     junit: {
       reportName: 'API Integration Tests'
     },
-    env: commonConfig.get('env'),
```

Comment:
Why remove this and the one in `test/functional/config`?

Reply:
I have the answer. The `env` value was removed from `test/common/config.js` because it used to have the kibana/es server specific stuff. These are left over in the `test/functional/config.js` and `test/api_integration/config.js`. It's part of the cleanup that didn't get removed earlier.

Reply:
Neither of these is part of this PR, but I did go ahead and remove `env`. Hopefully that isn't too confusing for future reviewers. :) Can you point to what you're talking about with `transform_deprecations`?